src/base/detoken.lua

```diff
@@ -49,6 +49,7 @@
 			-- convert the token into a function to execute
 			local func, err = loadstring("return " .. token)
 			if not func then
+				print("load error:", err)
```

Comment:
correct, although the error() would just display whatever is in "err" and give no context as to what caused that err... so I added a print here so I did get some information about whether it was the load that failed, or something else.